compiler/rustc_typeck/src/check/upvar.rs

```diff
@@ -447,7 +447,8 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
     ///       the existing min_capture map that is stored in TypeckResults.
     ///
     /// Eg:
-    /// ```rust,no_run
+    /// ```ignore HELP ("could not find Display in fmt", is there any way to make this work? I assume we don't have std at this point?)
+    /// #[derive(std::fmt::Display)]
```

Comment:
There is no `Display` derive macro. Only `Debug` can be derived.

Reply:
Right :facepalm: